index.md

```diff
@@ -0,0 +1,88 @@
+  We need people to help us organize or respond
+  </span>
+</a>
+
+Have something else you need? Please [open an issue](https://github.com/foss-responders/support-requests/issues/new), [join our Slack](https://slack.opencollective.com/#crisis-working-group) or email [foss-responders@googlegroups.com](mailto:foss-responders@googlegroups.com). If you'd rather email than open a GitHub issue, please feel free to email, as well.
+
+## Conferences
+
+- Here is [a dynamic list of cancelled events](https://airtable.com/shrETNURgXNrGWbd8/tblc49hMMykARebo8). Not seeing an event? [Add it here](https://airtable.com/shr5QBJUPPOQUJfND).
+- If you were sponsoring a cancelled conference for recruitment purposes and you still want to connect with prospective candidates, Indeed is working on scheduling virtual hiring events. [Click here for more information](https://events.indeed.com/hiringevents/#get_started_form).
+
+What else should be here? [Get in touch](mailto:foss-responders@googlegroups.com!)
+
+
+## Funding
+
+- [Indeed](https://indeed.com) is running another round with $10k to focus on projects impacted by COVID-19. They're looking for others to join in organization, sponsorship, mentorship, and outreach.
+
+What else should be here? [Get in touch](mailto:foss-responders@googlegroups.com).
+
+## About
+
+<p class="center"><a href="https://slack.opencollective.com/#crisis-working-group"><img src="https://img.shields.io/badge/slack-open%20collective-blue" alt="Slack link to Open Collective Slack"/></a></p>
+
+Open source software depends upon real-world interactions just as much as it does on code. With the alarming spread of the Coronavirus, the open source ecosystem is being affected in enormously impactful ways. This site is one of many efforts to help.
+
+It started as an organic movement led by Duane O'Brien (Indeed.com), Megan Byrd-Sanicki (Google), and others. We're open to all who can helping out!
+
+## Sponsors and Backers
+
+FOSS Responders is an open source, open project. Anyone can jump in and work on it. It is entirely volunteer run. Right now, we're in particular need of funders, designers, and people to spread the word.
+
+Some of the companies where our volunteers work (but may or may not be representing their company) include:
+
+- [GitLab](https://about.gitlab.com/)
+- [Google](https://google.com)
+- [Indeed](https://protocol.ai)
+- [Maintainer Mountaineer](https://maintainer.io)
+- [Open Source Initiative](https://opensource.org/)
+- [Salesforce](https://www.salesforce.com/)
+- [Storj](https://storj.io)
+- ...and yours?
+
+## Other Efforts
+
+While not dedicated entirely to open source, here are some other programs that need help
+
+* [Covid19 Response Fund](https://covid19responsefund.org/). Donate directly to the WHO, who will allocate funds to efforts that need it.
+* [Portland Corona Virus Mutual Aid Open Collective](https://opencollective.com/portland-corona-virus-mutual-aid-fund). Donate to those in Portland who need help.
+
```

Comment:
`* [Organizing Virtual Events Handbook](https://make.wordpress.org/community/handbook/meetup-organizer/resources/organizing-virtual-events/). Assembled by the Wordpress community.`